security/org-account-context/dependencies.tf

```diff
@@ -13,6 +13,17 @@ data "aws_iam_policy_document" "assume_role_policy_self" {
   }
 }
 
+data "aws_iam_policy_document" "assume_role_policy_selfservice" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::${var.shared_account_id}:role/aad-aws-sync"]
```

Comment:
Where does this role get created?

Reply:
It doesn't. Not yet anyways. It would have to be done separately. Since this repo is public, I'm not going to disclose it here.